content/root/best-practices-guide.textile

```diff
@@ -0,0 +1,97 @@
+---
+title: Best Practices Guide
+section: root
+index: 30
+jump_to:
+  Contents:
+    - Ably 101
+    - Using Ably like a pro
+---
+
+h2(#title). Ably Best Practices Guide
+
+In order to avail the best advantage of our platform, be sure to follow this best practices guide while building your applications using Ably.
+
+h3(#ably-101). Ably 101
+
+Ably Realtime is a cloud-based platform that allows you to easily integrate realtime functionality into your applications. Just plug and play.
+
+h4(#pubsub). Publish and Subscribe
+
+Publish data to a named "channel":/realtime/channels-messages on the Ably Realtime platform, and everyone subscribed to that same channel will receive it.
+
+h4(#notifications). Notify
+
+You can use "Push Notifications":/realtime/push to inform your users about the cool new features you are building in your application.
+
+h4(#presence). Be present
+
+Go online and let others in your application know this by using "Presence":/realtime/presence. You can also see who else is present on a channel even when you yourself are NOT present.
```

Comment:
They don't go online with presence, a client becomes present on a channel.  Again, why are talking about people as  opposed to devices or clients.  Presence could be used by an IoT device for example to indicate it is online and connected.  Why is `NOT` in capitals?  I am not sure of the relevance of explaining in this description that you can do that, this high level summary should explain what presence is and how it can be used.  Whether a client is present or not IMHO is not all that relevant or helpful at this point.